packages/data-explorer-ui/src/components/common/Banner/banner.styles.ts

```diff
@@ -0,0 +1,9 @@
+import styled from "@emotion/styled";
+
+export const Banner = styled.div`
+  background-color: ${({ theme }) => theme.palette.primary.main};
+  color: ${({ theme }) => theme.palette.common.white};
+  padding: 8px;
+  line-height: 1;
```

Comment:
```suggestion
```

Reply:
I think let's not try to control the line height from the parent component - the content should be driver for line height instead.